rosbag2_compression/CMakeLists.txt

```diff
@@ -89,6 +89,7 @@ install(
 
 ament_export_include_directories(include)
 ament_export_libraries(${PROJECT_NAME}_zstd)
+ament_export_libraries(${PROJECT_NAME})
```

Comment:
This can be simplified to `ament_export_libraries(${PROJECT_NAME} ${PROJECT_NAME}_zstd)`